pkg/api/vm/handler.go

```diff
@@ -562,6 +575,60 @@ func (h *vmActionHandler) copySecret(sourceName, targetName string, templateVers
 
 }
 
+func (h *vmActionHandler) createVMImages(templateVersion *harvesterv1.VirtualMachineTemplateVersion, vm *kubevirtv1.VirtualMachine) error {
+	for index, volume := range vm.Spec.Template.Spec.Volumes {
+		if volume.PersistentVolumeClaim == nil {
+			continue
+		}
+		pvc, err := h.pvcCache.Get(vm.Namespace, volume.PersistentVolumeClaim.ClaimName)
+		if err != nil {
+			return err
+		}
+		storageClassName := "harvester-longhorn"
```

Comment:
should get default sc manually as this is configurable by the users.

Reply:
Updated it. Thank you.